src/pybind/mgr/dashboard/frontend/src/app/shared/components/delete-confirmation-modal/delete-confirmation-modal.component.ts

```diff
@@ -70,6 +79,13 @@ export class DeleteConfirmationModalComponent extends BaseModal implements OnIni
     if (!(this.submitAction || this.submitActionObservable)) {
       throw new Error('No submit action defined');
     }
+    if (this.subsystem > 0) {
+      controls.confirmInput.disable();
+      this.setFlag = true;
```

Comment:
`setFlag`? what is it setting? we should be more verbose with the variable names so we know what it does from just looking at it